integration-cli/docker_cli_run_test.go

```diff
@@ -620,11 +696,16 @@ func (s *DockerSuite) TestRunEnvironmentOverride(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunContainerNetwork(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	dockerCmd(c, "run", "busybox", "ping", "-c", "1", "127.0.0.1")
+	if daemonPlatform == "windows" {
+		// Windows busybox does not have ping. Use built in ping instead.
+		dockerCmd(c, "run", WindowsBaseImage, "ping", "-n", "1", "127.0.0.1")
+	} else {
+		dockerCmd(c, "run", "busybox", "ping", "-c", "1", "127.0.0.1")
+	}
 }
 
 func (s *DockerSuite) TestRunNetHostNotAllowedWithLinks(c *check.C) {
+	// TODO Windows: --link is not yet functional.
```

Comment:
@jhowardmsft fyi, not 100% sure if you even need to get `--link` working on windows since we're moving towards `docker network` and `docker service`. /cc @mavenugo